docs/types.rst

```diff
@@ -54,7 +54,7 @@ Operators:
 * Bit operators: ``&``, ``|``, ``^`` (bitwise exclusive or), ``~`` (bitwise negation)
 * Arithmetic operators: ``+``, ``-``, unary ``-``, unary ``+``, ``*``, ``/``, ``%`` (remainder), ``**`` (exponentiation), ``<<`` (left shift), ``>>`` (right shift)
 
-Division always truncates (it just maps to the DIV opcode of the EVM), but it does not truncate if both
+Division always truncates (it just maps to the DIV opcode of the EVM. Maps will be explained later `[here](http://solidity.readthedocs.io/en/develop/types.html#mappings)`_), but it does not truncate if both
```

Comment:
The markup looks like a combination of Markdown and reStructuredText.  This document is written in reStructuredText.  Also, the URL should not appear here.  An older version of the document should not link to the newest version of the document like this.  Use `:ref` role to create a link.